io.c

```diff
@@ -12699,10 +12682,14 @@ rb_io_external_encoding(VALUE io)
 
 /*
  *  call-seq:
- *     io.internal_encoding   -> encoding
+ *    internal_encoding -> encoding or nil0
```

Comment:
I think there's a typo here:

```suggestion
 *    internal_encoding -> encoding or nil
```